styles/components/footer.css

```diff
@@ -0,0 +1,37 @@
+#connect {
+  min-height: 30vh;
+  color: white;
+  background: var(--COLOR-PRIMARY-XXDARK);
+  text-align: center;
+}
```

Comment:
just a question, why have you used id here ?

Reply:
I was thinking to scroll to the footer via clicking it on the header(connect button) but, didn't add that functionality though. Will change it back to class